site-src/geps/gep-1424.md

```diff
@@ -0,0 +1,384 @@
+
+## Personas
+
+This GEP uses the [roles and personas](https://gateway-api.sigs.k8s.io/concepts/security-model/#roles-and-personas) defined in the Gateway API security model.
+
+## Goals
+
+* SHOULD allow `HTTPRoute` traffic rules to be configurable for a mesh service by the application owner/producer.
+* SHOULD allow control by the cluster operator (mesh administrator) to grant permission for whether `HTTPRoute` resources in a given namespace are allowed to configure mesh traffic routing.
+* MUST only be allowed to configure "producer" traffic rules for a `Service` in the same namespace as the `HTTPRoute`.
+    * Traffic routing configuration defined in this way SHOULD be respected by ALL consumer services in all namespaces in the mesh.
+* SHOULD only be allowed to direct traffic to `backendRefs` within the same namespace when used for configuration of mesh traffic.
+    * This constraint may be revisited in the future, but is intended to help simplify an initial implementation.
+    * Attempting to specify an `HTTPBackendRef` in a different namespace should raise a status condition on the `HTTPRoute`.
+    * When an `HTTPBackendRef` is invalid, 500 status codes MUST be returned for requests that would have otherwise been routed to an invalid backend.
+        * This matches existing behavior as defined in [`HTTPRouteRule`](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io%2fv1beta1.HTTPRouteRule).
+* SHOULD set a status field on `HTTPRoute` to show if the routing configuration has been applied to the mesh.
+* MAY assume that a mesh implements "transparent proxy" functionality to redirect calls to the Kubernetes DNS address for a `Service` through mesh routing rules.
+    * For all namespaces configured by a cluster operator in the scope of a mesh deployment.
+* SHOULD NOT require reconfiguring existing `HTTPRoutes` for North/South Gateway configuration.
+
+## Non-Goals
+
+* Supporting "egress" use cases, which is currently a deferred goal, including:
+    * Defining how "consumer" traffic rules which could override routing for service upstreams only within the local scope of a namespace or service might be configured.
+    * Redirecting calls from arbitrary custom domains to an in-cluster service.
+* Defining how multiple `Services` or `EndpointSlices` representing instances of a single "logical" service should present an identity for AuthN/AuthZ or be associated with each other beyond routing rules.
+* Defining how AuthZ should be implemented to secure East/West traffic between services.
+* Defining how [Policy Attachment](https://gateway-api.sigs.k8s.io/references/policy-attachment/) would bind to `HTTPRoute`, services or a mesh.
+* Defining how `HTTPRoutes` configured for East/West service mesh traffic management might integrate with North/South `Gateways`.
```

Comment:
```suggestion
* Defining how `HTTPRoutes` configured for East/West service mesh traffic management might integrate with North/South `Gateways`. This will be defined in a future GEP.
```
Opened https://github.com/kubernetes-sigs/gateway-api/issues/1478 to track this, should it be linked here explicitly @robscott?